app/controllers/api/invitations_controller.rb

```diff
@@ -28,8 +40,4 @@ def invitation_parser
     @invitation_parser ||= InvitationParser.new(@invitations)
   end
 
-  def resource_serializer
```

Comment:
Hey this is going to break the existing invitation system pretty badly, you may need to rethink how you're serializing these, or allow the new serializer to handle both cases